docs/enterprise/install/readme.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Install
+lang: en-US
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy oidc docker reverse-proxy containers install enterprise console
+---
+
+There are several ways to install Pomerium Enterprise, to suite your organization's needs. [Let us know] if you don't see an installation method compatible with your infrastructure.
```

Comment:
We should mention that we do support binaries, rpms, debs, and a wide range of deployment options.